packages/dds/ink/src/interfaces.ts

```diff
@@ -56,8 +56,9 @@ export interface IColor {
 }
 
 export interface IInkEvents extends ISharedObjectEvents {
-    (event: "stylus", listener: (operation: IStylusOperation) => void);
-    (event: "clear", listener: () => void);
+    stylus: [operation: IStylusOperation];
+    clear: [operation: IClearOperation];
+    createStroke: [operation: ICreateStrokeOperation];
```

Comment:
These event typings were mismatched/wrong before, and gave an error in the `emit` calls with my changes. Corrected to include the operation in `clear` and added the `createStroke` event.